test/cluster/podspec_test.go

```diff
@@ -532,15 +573,16 @@ func getEnvVar(envVar string) string {
 	return envVarVal
 }
 
-func validateImageRegistry(
-	k8sClient client.Client, _ goctx.Context, aeroCluster *asdbv1.AerospikeCluster, registry string,
-) {
+func validateImageRegistryNamespace(k8sClient client.Client, _ goctx.Context, aeroCluster *asdbv1.AerospikeCluster,
```

Comment:
```suggestion
func validateInitImage(k8sClient client.Client, _ goctx.Context, aeroCluster *asdbv1.AerospikeCluster,
```

Remove unused context